README.md

```diff
@@ -1456,8 +1456,8 @@ You can change the criteria for being considered the `best` by using `-S` (`--fo
 
 The available fields are:
 
- - `hasvid`: Gives priority to formats that has a video stream
- - `hasaud`: Gives priority to formats that has a audio stream
+ - `hasvid`: Gives priority to formats that have a video stream
+ - `hasaud`: Gives priority to formats that have a audio stream
```

Comment:
have an

Reply:
I did that on purpose, because I think it looks better if lines 1459 and 1460 are the same length

Reply:
come to think of it, I'm not even sure if this should be changed from `has` to `have`, given the field names are `hasvid` and `hasaud`

Reply:
Yes, I'm agreeing with you on that here.